docs/tutorials/cups/index.md

```diff
@@ -0,0 +1,60 @@
+# Control and User Plane (CUPS) Separation
+
+In this tutorial, we will deploy and run the SD-Core 5G core network using Juju. We will also deploy a radio and cellphone simulator to simulate usage of this network.
```

Comment:
I think we're missing the simulation part. Is this still WIP?

Reply:
Added